### PR TITLE
(+) custom cy.login command

### PR DESCRIPTION
**Pull request by r-wasserman**
Custom cypress command to perform login through UI.

### CODE REVIEW

File: cypress/support/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/// <reference types="cypress" />
+declare namespace Cypress {
+    interface Chainable<Subject = any> {
```
Comment (JoelForamitti) [inline cypress/support/index.d.ts:3]: Here it gives a warning: `'Subject' is defined but never used`

**Review by JoelForamitti — CHANGES_REQUESTED** [review]

File: cypress/support/index.d.ts (hunk 1 above)
Comment (r-wasserman) [reply cypress/support/index.d.ts]: Hello @JoelForamitti, 
I adjusted the `intex.d.ts` file accordingly. I can't seem to find the warning locally, this is a first try to solve it. 
Let me know!

**Review by JoelForamitti — APPROVED** [review]

File: cypress/support/index.d.ts (hunk 1 above)
Comment (JoelForamitti) [reply cypress/support/index.d.ts]: Thanks. You can find the warning by running `yarn lint`

Comment (r-wasserman) [reply cypress/support/index.d.ts]: I always something new!

**Pull request merged**